package.json

```diff
@@ -119,17 +119,20 @@
     "http-errors": "^2.0.0",
     "jquery": "^3.7.1",
     "jwt-decode": "^4.0.0",
+    "moment": "^2.30.1",
     "nocache": "^4.0.0",
     "nunjucks": "^3.2.4",
     "passport": "^0.7.0",
     "passport-oauth2": "^1.7.0",
     "prom-client": "^15.1.0",
     "redis": "^4.6.12",
     "superagent": "^8.1.2",
+    "universal-analytics": "^0.5.3",
```

Comment:
we can remove this dependency since we're not gonna raise analytics events server-side

(ps: google analytics 4 has replaced google universal analytics)

Reply:
we don't need this dependency